config/mail/content.ts

```diff
@@ -33,11 +37,18 @@ export function resolveContent({
   }
 
   if (html) {
-    newContent.push({
+    const newMime: Content = {
       mimeType: 'text/html; charset="utf-8"',
       content: quotedPrintableEncode(html),
       transferEncoding: "quoted-printable",
-    });
+    };
+
+    if (relatedAttachments) {
+      newMime.relatedAttachments = relatedAttachments.map((attachment) =>
```

Comment:
put this call directly in line 43/44 inside the object. And revert to the direct call of push

Reply:
relatedAttachements is allways an array (see below)